studybuddies/src/app/components/error-page/error-page.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-error-page',
+  templateUrl: './error-page.component.html',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
  templateUrl: "./error-page.component.html",
```